File: es6/E001.js
helpers = require('./helpers')



var solution = function() {

    var max = 1000;

    var sum = 0;
    for (x of helpers.ascendingRangeGen(0, 1001, 1)) {

        if (x % 3 == 0 || x % 5 == 0) {
            sum += x;
        }

    };

    return sum;

}

File: python/E001.py

def set_multiples_to_x(x, multiplier):
    return set(xrange(0, x, multiplier))

def solution():
    threes = set_multiples_to_x(1000, 3)
    fives = set_multiples_to_x(1000, 5)
    threes_and_fives = threes.union(fives)
    return sum(threes_and_fives)